### ISSUE TITLE
How to stop html bibliography coming out grey?

### ISSUE THREAD

**Issue by abchenson** [mcmtroffaes/sphinxcontrib-bibtex#130]
Using default style options (or my own alterations based on https://sphinxcontrib-bibtex.readthedocs.io/en/latest/usage.html) my .rst file containing the  .. bibliography:: directive comes out greyed out.  It would be much better for me to have the text appear in black but I don't see how to achieve that with the style options.  Is this possible?
Many thanks

![sphinx_ref_screenshot](https://user-images.githubusercontent.com/32451279/31995633-3437f376-b97d-11e7-96bb-8f740f058c89.png)

**Comment by mcmtroffaes** [comment]
Well, formatting is done by sphinx, and has nothing to do with the extension per se. I'd suggest you look at the html theming support in sphinx: http://www.sphinx-doc.org/en/stable/theming.html If you inspect the generated html, hopefully you can figure out which options in your current theme lead to the gray text, and then change it according to your own preferences. You could also simply switch to another theme and see if it looks better for you. I hope this is helpful!

**Comment by utensil** [comment]
I encountered a similar problem and found this issue. My version of problem is local build looks pretty and fine:

![image](https://user-images.githubusercontent.com/64258/58624308-89685480-8302-11e9-9466-a5cc206c9940.png)

while the build by readthedocs.io used a different style for the label and the bibliography is also grey, just like the snapshot in the main thread. Also I observed that the HTML structure has also changed, from definition list for local build to table.

Eventually I figured out why by noticing https://github.com/rtfd/sphinx_rtd_theme/issues/752 and I specified `sphinx>=1.4` in `requirements.txt` for readthedocs and local sphinx is 2.0.

Hope it helps for anyone who may encounter a similar problem.
